PluginBuilder/Views/Plugin/Settings.cshtml

```diff
@@ -33,10 +34,59 @@
                 <label asp-for="BuildConfig" class="form-label"></label>
                 <input asp-for="BuildConfig" class="form-control" placeholder="Release" />
                 <span asp-validation-for="BuildConfig" class="text-danger"></span>
+            </div> 
+            <div class="form-group">
+                <label class="form-label">Select Tags</label>
+                <div class="dropdown">
+                    <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-bs-toggle="dropdown" aria-expanded="false">
+                        Select Tags
+                    </button>
+                    <ul class="dropdown-menu p-3" aria-labelledby="dropdownMenuButton">
+                        @foreach (var tag in Enum.GetValues(typeof(PluginTags)).Cast<PluginTags>())
+                        {
+                            <li class="form-check">
+                                <input type="checkbox" class="form-check-input category-checkbox" value="@((int)tag)" id="tag@(tag)"
+                                       name="Tags" @(Model.PluginTags != null && Model.PluginTags.Any(t => t.Name == tag.ToString()) ? "checked" : "")>
+                                <label class="form-check-label" for="tag@(tag)">@tag</label>
+                            </li>
+                        }
+                    </ul>
+                </div>
+                <span asp-validation-for="PluginTags" class="text-danger"></span>
+                <input type="hidden" name="SelectedCategories" id="SelectedCategories">
             </div>
+
             <div class="form-group mt-4">
                 <input type="submit" value="Update" class="btn btn-primary" id="Update" />
             </div>
         </form>
     </div>
 </div>
+
+
+<script>
+    document.addEventListener('DOMContentLoaded', function () {
+        var checkboxes = document.querySelectorAll('.category-checkbox');
+        var selectedCategoriesInput = document.getElementById('SelectedCategories');
+        var dropdownButton = document.getElementById('dropdownMenuButton');
+        function updateButtonLabel() {
+            var selectedCategories = [];
+            checkboxes.forEach(function (cb) {
+                if (cb.checked) {
+                    selectedCategories.push(cb.value);
+                }
+            });
+            selectedCategoriesInput.value = selectedCategories.join(',');
+
+            if (selectedCategories.length > 0) {
+                dropdownButton.textContent = selectedCategories.length + ' tag' + (selectedCategories.length > 1 ? 's' : '') + ' selected';
+            } else {
+                dropdownButton.textContent = 'Select tags';
+            }
+        }
+        checkboxes.forEach(function (checkbox) {
+            checkbox.addEventListener('change', updateButtonLabel);
+        });
+        updateButtonLabel();
+    });
```

Comment:
You can put this in a separate JS file to avoid duplication. You could also encapsulate the view part above in a  view component and eliminate that duplication too this way.

Reply:
Decided to remove that, and use just input for tags using tom-select.

You can go ahead and review